addons/netfox.extras/state-machine/rewindable-state-machine.gd

```diff
@@ -133,6 +159,10 @@ func _get_configuration_warnings():
 func _rollback_tick(delta: float, tick: int, is_fresh: bool) -> void:
 	if _state_object:
 		_state_object.tick(delta, tick, is_fresh)
+		
+		print((_state_object.get_script() as Script).source_code)
+		print(_state_object.get_signal_list().map(func(it): return it["name"]))
```

Comment:
Not sure why doing loggings here. 
Maybe we could do these stuffs in debug mode. 
I am not sure which function could check that.

Reply:
It's leftover code from a debugging session, will be removed